app/components/emoji/emoji.js

```diff
@@ -49,10 +64,16 @@ export default class Emoji extends React.PureComponent {
             return <Text style={textStyle}>{literal}</Text>;
         }
 
+        let ImageComponent = FastImage;
+        if (Platform.OS === 'android') {
```

Comment:
After looking at the fastImage repo I think we can use FastImage for android as well don't you think?